src/user-scripts/deploy-pool.ts

```diff
@@ -32,7 +53,7 @@ const reserve_configs: ReserveConfig[] = [
     l_factor: 980_0000,
     util: 900_0000, //must be under 950_0000
     max_util: 980_0000, //must be greater than util
-    r_base: 100,
+    r_base: 1001,
     r_one: 50_0000,
     r_two: 100_0000,
     r_three: 1_000_0000,
```

Comment:
these values are denominated in SCALAR_7 - this looks like it could be confused with BPS.

Can we update to:

```
        r_base: 0_0050000, // 0.5%
        r_one: 0_0500000, // 5%
        r_two: 0_1000000, // 10%
        r_three: 1_0000000, // 100%
```

Reply:
Typescript won't allow this it errors with `Numeric separators are not allowed`